bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -435,7 +499,12 @@ async def post_review(self, ctx: Context, user_id: int) -> None:
     @Cog.listener()
     async def on_member_ban(self, guild: Guild, user: MemberOrUser) -> None:
         """Remove `user` from the talent pool after they are banned."""
-        await self.end_nomination(user.id, "User was banned.")
+        if await self.end_nomination(user.id, "Automatic removal: User was banned"):
+            nomination_discussion = await get_or_fetch_channel(Channels.nomination_discussion)
+            await nomination_discussion.send(
```

Comment:
The implementation of `get_or_fetch_channel` makes me generally uncomfortable since there's no error handling. You'll be bombarded with errors if say something happens to the channel.